pkg/controller/vsphere/machine_scope_test.go

```diff
@@ -547,7 +547,8 @@ func TestNodeGetter(t *testing.T) {
 	testEnv := &envtest.Environment{
 		CRDDirectoryPaths: []string{
 			filepath.Join("..", "..", "..", "install"),
-			filepath.Join("..", "..", "..", "vendor", "github.com", "openshift", "api", "config", "v1", "zz_generated.crd-manifests")},
+			filepath.Join("..", "..", "..", "vendor", "github.com", "openshift", "api", "config", "v1", "zz_generated.crd-manifests"),
+			filepath.Join("..", "..", "..", "third_party", "cluster-api", "crds")},
```

Comment:
What new CRDs are needed?

Reply:
I have gone through and removed this one entry but the other locations are still needed.  I think I added this here as part of my original debug.